pkg/network/protocols/http/driver_interface.go

```diff
@@ -28,16 +28,30 @@ const (
 	httpReadBufferCount = 100
 )
 
-type FullHttpTransaction struct {
+type WinHttpTransaction struct {
 	Txn             driver.HttpTransactionType
 	RequestFragment []byte
+
+	// ... plus some extra that's only valid when it's an ETW transactoin
+	AppPool string
+
+	// <<<MORE ETW HttpService DETAILS>>>
+	// We can track FULL url and few other attributes. However it will require much memory.
+	// Search for <<<MORE ETW HttpService DETAILS>>> top find all places to be uncommented
+	// if such tracking is desired
+	//
```

Comment:
You probably can remove this comment by now. I left it only as a breadcrumbs for very next guy, and you are already know that fact.